python/python/glide/async_commands/cluster_commands.py

```diff
@@ -521,40 +528,40 @@ async def sort(
         See https://valkey.io/commands/sort for more details.
 
         Args:
-            key (str): The key of the list, set, or sorted set to be sorted.
+            key (TEncodable): The key of the list, set, or sorted set to be sorted.
             limit (Optional[Limit]): Limiting the range of the query by setting offset and result count. See `Limit` class for more information.
             order (Optional[OrderBy]): Specifies the order to sort the elements.
                 Can be `OrderBy.ASC` (ascending) or `OrderBy.DESC` (descending).
             alpha (Optional[bool]): When `True`, sorts elements lexicographically. When `False` (default), sorts elements numerically.
                 Use this when the list, set, or sorted set contains string values that cannot be converted into double precision floating point numbers.
 
         Returns:
-            List[str]: A list of sorted elements.
+            List[bytes]: A list of sorted elements.
 
         Examples:
-            >>> await client.lpush("mylist", '3', '1', '2')
-            >>> await client.sort("mylist")
-            ['1', '2', '3']
+            >>> await client.lpush(b"mylist", b'3', b'1', b'2')
+            >>> await client.sort(b"mylist")
```

Comment:
```suggestion
            >>> await client.sort("mylist")
```

same reason, lets have mixed examples